src/tox_travis/after.py

```diff
@@ -81,10 +80,9 @@ def travis_after():
     print('All required jobs were successful.')
 
 
-def after_config_matches():
+def after_config_matches(cfg):
     """Determine if this job should wait for the others."""
-    config = py.iniconfig.IniConfig('tox.ini')
-    section = config.sections.get('travis:after', {})
+    section = cfg.sections.get('travis:after', {})
```

Comment:
I don't particularly like passing the tox config all the way down the call stack, but it seems kind of necessary in the current design. Other options would be to:
- set a module-level global that `after_config_matches` accesses directly.
- create a `ToxAfter` class that takes the config in it's init, then refactor the various module functions as methods, and `after_config_matches` accesses the `config` directly on the instance.

But I'm not convinced that these options are a clear improvement. Just ideas.